pkg/ccl/importccl/import_stmt.go

```diff
@@ -696,67 +736,66 @@ func (r *importResumer) Resume(
 // stuff to delete the keys in the background.
 func (r *importResumer) OnFailOrCancel(ctx context.Context, txn *client.Txn) error {
 	details := r.job.Details().(jobspb.ImportDetails)
-	if details.BackupPath != "" {
```

Comment:
can `BackupPath` be removed from the .proto?

Reply:
Eh, I realized that for now I'd like to keep it and assert it is empty to avoid adopting a `transform` job on a node that doesn't support `transform`